modules/rostests/apitests/win32nt/ntuser/NtUserSystemParametersInfo.c

```diff
@@ -583,8 +583,10 @@ Test_SPI_SETDESKWALLPAPER(void)
     RtlInitUnicodeString(&ustrOld, szOld);
 
     /* Set no Wallpaper */
+#ifdef _M_AMD64
```

Comment:
In the `ntgdi/NtGdiCreateBitmap.c` test above you used `_WIN64` (which means, _any_ 64-bit version, be it for AMD64 or ARM64...).
In this `ntuser/NtUserSystemParametersInfo.c` test here, is it expected to narrow it down to just `_M_AMD64` ?

In addition, why should this `SPI_SETDESKWALLPAPER` test be only done on AMD64, and not on x86?

Reply:
Sorry, I don't have an arm64 device to test that. 
That was a typo. It should NOT be run on _M_AMD64, because it's failing there. Fixed it.